tools/run_tests/performance/build_performance.sh

```diff
@@ -43,6 +43,9 @@ do
   "go")
     tools/run_tests/performance/build_performance_go.sh
     ;;
+  *"php"*)
+    tools/run_tests/performance/build_performance_php.sh $language $CONFIG
```

Comment:
Also, looks like you are now building php twice - once for php7 and once for php7_protobuf_c, given the build process seems to be the same regardless of using protobuf_c, maybe you can just build everything as language "php"?

Basically, that would mean differentiating the language name in scenario_config.py ( = `str(language)`) and "language to build" by introducing   `language.lang_to_build()`.

Building PHP twice is not a problem if:
1.) the second build takes negligible time (everything already built and installed)
2.) you can guarantee that the PHP builds won't run concurrently and therefore collide with each other.

Reply:
so the builds are not running concurrently as they are executed one by one by build_perfomance.sh

One possible trick (and possibly the simplest):
if both php7 and php7_protobuf_c are passed to build_performance.sh, 
it can remember that php has already been build when building the first build is triggered
(PHP_ALREADY_BUILT=the_lang_that_built_it), and the second request to build can be skipped (`"Skipping PHP build as already built by $PHP_ALREADY_BUILT"`).

Reply:
Done. Thank you for the suggestions. I check PHP_ALREADY_BUILT and make it build only once.